react-frontend-app/src/App.js

```diff
@@ -11,7 +11,6 @@ import logo from './logo.svg';
 import './style/App.css';
 
 // Import different components
-import Description from './components/Description'
```

Comment:
Any reason why this is booted? I think it's chill to not have Description as a separate component, but just to be on the same page.

Reply:
oh yeah i meant to delete all the description files since we could just paste the fat paragraph into the app.js

Reply:
Sounds good, unsure if we will need it again down the road but that works for me.  Ideally there would be an easy place for Zest to access and edit, if we can make it as clear as possible that would be optimal